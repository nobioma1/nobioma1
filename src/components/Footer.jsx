import React from 'react';
import { FooterContainer } from '../styles';

const Footer = () => {
  return (
    <FooterContainer>
      <p>Noble Obioma &copy; 2019</p>
    </FooterContainer>
  );
};

export default Footer;
