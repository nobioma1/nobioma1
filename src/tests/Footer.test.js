import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../components/Footer';

describe('Footer', () => {
  it('Footer Should Match Snapshot', () => {
    const tree = renderer.create(<Footer />);

    expect(tree).toMatchSnapshot();
  });
});
