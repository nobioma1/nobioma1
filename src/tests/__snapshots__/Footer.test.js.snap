// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer Should Match Snapshot 1`] = `
<div
  className="sc-kAzzGY iedvNq"
>
  <p>
    Noble Obioma Â© 2019
  </p>
</div>
`;
